packages/sdk/tasks/one-time/jarvis-polygon-plugins.ts

```diff
@@ -0,0 +1,104 @@
+import { task, types } from "hardhat/config";
+
+const POOL_NAME = "Jarvis jFIAT";
+const COMPTROLLER = "0xD265ff7e5487E9DD556a4BB900ccA6D087Eb3AD2";
+
+const UNDERLYINGS = {
+  "agEUR-jEUR": "0x2ffbce9099cbed86984286a54e5932414af4b717",
```

Comment:
Don't particularly love having the underlyings here as we already have them all in `assets/polygon.ts`, as well as the symbols, but for now that's alright I guess

Reply:
100% 
true true, maybe good to import them from polygon. But yes it's a one time shot. So maybe not worth the effort if they are at least the same :D